config_models/admin.py

```diff
@@ -128,7 +128,7 @@ def has_output(self):
         """ Should this filter be shown? """
         return True
 
-    def choices(self, cl):
+    def choices(self, cl):  # pylint: disable=arguments-differ
```

Comment:
@e-kolpakov not ignoring at all. Sorry about this one.

I thought you only meant this on the `edx-enterprise` repository, which is where you wrote that comment.

I'll make the interface consistent.